client/src/main/java/com/paypal/selion/reports/reporter/runtimereport/JsonRuntimeReporterHelper.java

```diff
@@ -83,7 +83,8 @@
     public JsonRuntimeReporterHelper() {
         try {
             File workingDir = new File(Config.getConfigProperty(ConfigProperty.WORK_DIR));
-            workingDir.mkdirs();
+            boolean isCreated = workingDir.mkdirs();
+            logger.log(Level.FINER,"Is working directory creation successful:"+ isCreated);
```

Comment:
@mach6  yes, it is not used anywhere. I added this to overcome the violation and also it made sense to look this up in the log while debugging. I will reword this.